kuma/users/stripe_utils.py

```diff
@@ -95,6 +95,26 @@ def create_stripe_customer_and_subscription_for_user(user, email, stripe_token):
     UserSubscription.set_active(user, subscription.id)
 
 
+def cancel_stripe_customer_subscriptions(user):
+    """Delete all subscriptions for a Stripe customer."""
+    assert user.stripe_customer_id
+    cancelled = []
+    for subscription_id in cancel_stripe_customer_subscription(user.stripe_customer_id):
+        UserSubscription.set_canceled(user, subscription_id)
+        cancelled.append(subscription_id)
+    return cancelled
+
+
+def cancel_stripe_customer_subscription(stripe_customer_id):
```

Comment:
I'd reconsider renaming those two fns, imo they don't differ on the fact they operate on single/multiple subscriptions as the names imply, but that one does it over at stripe, and one uses that to deactive our local ones. Here's an idea that's probably not optimal either: `cancel_stripe_subscriptions` and `cancel_subscriptions`. Where the latter calls the former and keeps up the pretense that we could switch out subscription service providers.

Reply:
Gee. I can't remember why `cancel_stripe_customer_subscription` even exists! The `api.v1` needs only the `cancel_stripe_customer_subscriptionS` one. Where or which place, outside, this file, needs to dig in specifically to `cancel_stripe_customer_subscription`??